src/main/kotlin/org/dif/common/TYP.kt

```diff
@@ -0,0 +1,7 @@
+package org.dif.common
+
+enum class TYP(typ: String) {
```

Comment:
Is TYP an acronym? If not I'd suggest upper camel case, and if it's short for `type` I'd suggest prefixing it with something for context